src/reducers/multicall/feeds.js

```diff
@@ -76,10 +47,7 @@ export const flipper = addresses => name => ({
   call: ['ilks(bytes32)(address,uint256,uint256)', toHex(name)],
   returns: [
     [`${name}.${LIQUIDATOR_ADDRESS}`],
-    [
-      `${name}.${LIQUIDATION_PENALTY}`,
-      val => fromRay(mul(sub(val, RAY), 100), 2)
-    ],
+    [`${name}.${LIQUIDATION_PENALTY}`],
```

Comment:
here for example i would probably change this to
```js
[`${name}.${LIQUIDATION_PENALTY}`, chop => liquidationPenalty(chop)]
```
where `liquidationPenalty` is
```js
import { liquidationPenalty } from '@makerdao/dai-plugin-mcd/dist/math';
```